jenkins/ci.suse.de/cloud-mkcloud8.yaml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: cloud-mkcloud8-gating
+    version: 8
+    previous_version: 7
+    label: cloud-trigger
+    jobs:
+        - 'cloud-mkcloud{version}-gating'
+- project:
+    name: cloud-mkcloud8-x86_64
+    disabled: false
+    version: 8
+    previous_version: 7
+    arch: x86_64
+    tempestoptions: --smoke
+    label: openstack-mkcloud-SLE12-x86_64
+    jobs:
+        - 'cloud-mkcloud{version}-job-2nodes-{arch}'
+        - 'cloud-mkcloud{version}-job-4nodes-linuxbridge-{arch}'
+          # - 'cloud-mkcloud{version}-job-backup-restore-{arch}'
+          # - 'cloud-mkcloud{version}-job-btrfs-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar-devsetup-{arch}'
+          #- 'cloud-mkcloud{version}-job-dvr-{arch}'
+          #- 'cloud-mkcloud{version}-job-magnum-{arch}'
+          #- 'cloud-mkcloud{version}-job-monasca-{arch}'
+          #- 'cloud-mkcloud{version}-job-raid-{arch}'
+          #- 'cloud-mkcloud{version}-job-ssl-{arch}'
+          #- 'cloud-mkcloud{version}-job-upgrade-disruptive-{arch}'
+          #- 'cloud-mkcloud{version}-job-xen-{arch}'
+- project:
+    name: cloud-mkcloud8-ha-x86_64
+    disabled: true
+    version: 8
+    previous_version: 7
+    arch: x86_64
+    tempestoptions: --smoke
+    label: openstack-mkcloud-ha-x86_64
+    jobs:
+        - 'cloud-mkcloud{version}-job-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-compute-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-linuxbridge-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-ironic-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-ceph-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-ceph-compute_ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-dvr-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-lbaasv2-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-without-nodes-upgrade-{arch}'
```

Comment:
I do not think there's much value in keeping these upgrade jobs here (even with `disabled: true`). What about just enabling the project with one simple HA job?